### PR TITLE
Submissions Model

### PR DESCRIPTION
**Pull request by nkoul**
Submissions added to App using built in rails microposts

### CODE REVIEW

**Comment by nkoul** [comment]
It was something I had seen in a rails tutorial and I had thought that it would provide all of the things we would need for a submissions model to work.

**Comment by nkoul** [comment]
I also just realized that rails has a submissions model that does pretty much the same thing, i'm going to go ahead and switch over to that (it seems to have the same functionality as microposts)

File: config/routes.rb [hunk 1]
```diff
@@ -1,12 +1,16 @@
 Rails.application.routes.draw do
-    resources :account_activations, only: [:edit] do
-    end
-    default_url_options :host => "localhost"
+  resources :submissions
+  root :to =>'sessions#new'
+  resources :submissions, :path => 'posts'
```
Comment (bethanyhaubert) [inline config/routes.rb]: You don't need to rename anything here. Using `submissions` is fine.

File: db/schema.rb [hunk 2]
```diff
@@ -11,7 +11,18 @@
...
+
+  create_table "submissions", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string   "category"
+    t.string   "title"
+    t.string   "content"
+    t.string   "user_id"
+    t.string   "upvotes"
+    t.string   "donwvotes"
```
Comment (bethanyhaubert) [inline db/schema.rb]: go ahead and remove `upvotes` and `downvotes` from this table

**Pull request merged**